account_fiscal_position_autodetect_optional_vies/readme/USAGE.rst

```diff
@@ -0,0 +1,10 @@
+Create a new partner and define Spain in country field.
+
+Create a new Fiscal Position, with the following configuration:
+
+* 'Detect Automatically' -> checked.
+* 'Vat required' -> checked.
+* 'Vat VIES required' -> checked.
+* 'Country group' -> Europe.
```

Comment:
Question: as far as I can see, this is already the default behavior - VIES check only applies to EU countries.
What is the benefit of this feature?

Reply:
Sorry for the confusion. It is not necessary to define any group of countries or country in the fiscal position, it will only be applied if the contact has a valid VIES.

Reply:
So 'Vat VIES required' means 'Vat was VIES validated'?  - Fiscal Position only applies if VAT passed VIES validation.

Reply:
Exactly, I think it is better to change the string so that it is better understood. Now I will change it.